test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -440,6 +440,8 @@ class PBSTestSuite(unittest.TestCase):
     scheds = {}
     moms = None
     comms = None
+    users = ""
+    added_users = []
```

Comment:
I don't think you need 2 separate variables to store a username and its role. It can be done by one variable also.

Reply:
Also, I suggest renaming `added_users` to something like `added_mgrs_oprs` for more readability